.github/workflows/cicd.yaml

```diff
@@ -37,50 +37,64 @@ jobs:
     needs: [ci]
     environment:
       name: devnet
-      url: 'http://app.dev.credix.finance'
+      url: https://app.dev.credix.finance
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0
+      - name: Install and build
+        run : |
+          yarn install --silent
+          yarn global add react-scripts@3.4.1 --silent
```

Comment:
why this version? it's not the one we use

Reply:
it's the same as in the Dockerfile, I guess it should work if we remove it, will try that